src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/wwwroot/css/open-iconic/README.md

```diff
@@ -0,0 +1,114 @@
+[Open Iconic v1.1.1](http://useiconic.com/open)
```

Comment:
We really need to get rid of this 🤮

Reply:
If only somebody had been saying we should update the template designs.

Reply:
To make matters worse all links are `HTTP` and point to a domain that seems to redirect to domain reseller that kind of sells iconography tools as a sidehustle. I am not sure if they are associated but the entire github org behind seems to be dead and there are already reports of hijacking in other libraries from that org:


site itself being hijacked for malware: https://github.com/mdo/ama/issues/319
another library from that org seems to have been through a sketchy history itself: https://github.com/iconic/SVGInjector/issues/63

Reply:
Filed a separate issue to track this https://github.com/dotnet/aspnetcore/issues/48341